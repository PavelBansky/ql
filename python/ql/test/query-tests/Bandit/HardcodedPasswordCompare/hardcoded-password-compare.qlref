Bandit/hardcoded-password-compare.ql