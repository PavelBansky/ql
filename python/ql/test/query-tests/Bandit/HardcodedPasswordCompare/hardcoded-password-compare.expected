| Test.py:5:8:5:25 | Compare | Possible hardcoded password |
| Test.py:9:8:9:21 | Compare | Possible hardcoded password |
| Test.py:13:8:13:43 | Compare | Possible hardcoded password |
